nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/writer/FileBaseWriter.scala

```diff
@@ -36,15 +37,18 @@ class NebulaSSTWriter(path: String) extends Writer {
   override def prepare(): Unit = {
     writer = new SstFileWriter(env, options)
     writer.open(path)
+    isOpen = true
   }
 
   def write(key: Array[Byte], value: Array[Byte]): Unit = {
     writer.put(key, value)
   }
 
   override def close(): Unit = {
-    writer.finish()
-    writer.close()
+    if (isOpen) {
```

Comment:
Because `prepare` has been called, `isOpen` must be `true` ...

Reply:
if `writer.open` failed， `isOpen` is still `false`.  Only `SstFileWriter` open successfully, `writer` can be closed.